sdk/scheduler/src/test/resources/test-update-pod-a.yml

```diff
@@ -0,0 +1,36 @@
+name: "test"
```

Comment:
Something to think about: dealing with minor changes between these files for tests. As it stands it's not easy to tell what's different between these files at a glance. Was thinking patch files but those would be annoying to maintain as the base files are updated. Maybe have a single common `RawServiceSpec` that gets parsed in from a single file, and then the `RawServiceSpec` is what gets modified for the tests?